doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po

```diff
@@ -597,6 +615,9 @@ msgid ""
 "object is fresh enough to serve. If it is fresh, then Traffic Server serves "
 "it to the client as a *cache hit* (see the figure below)."
 msgstr ""
+"オブジェクトがキャッシュにある場合は、Traffic Serverはオブジェクトが提供する"
+"のに十分新鮮かどうかを確認します。新鮮な場合、Traffic Serverはそれを*キャッ"
```

Comment:
Whitespace is required before start-string of inline markup

> 1. The start-string must start a text block or be immediately preceded by whitespace or any of  ' " ( [ { or <.
http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup